### PR TITLE
Reusable template Blade Shape Banner

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mayuran-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

File: src/features/components/templates/banners/blade-shape-banner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+import Container from 'features/components/atoms/container'
+import Flex from 'features/components/atoms/flex-box'
+import FlexBox from 'features/components/atoms/flex-box'
+import Image from 'features/components/atoms/image'
+import Typography from 'features/components/atoms/typography'
+import Link from 'features/components/atoms/link'
+import { Localize } from 'components/localization'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+
+const BladeShapeBanner = ({ data }: { data: BannerType }) => {
```
Comment (aswathy-deriv) [inline src/features/components/templates/banners/blade-shape-banner/index.tsx:13]: can you put this as an interface?`{ data: BannerType }`

File: src/features/components/templates/banners/blade-shape-banner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    return (
+        <Container.Fixed
+            className={styles.banner_section}
+            as={'section'}
+            bgcolor={'black'}
+            padding_block={'32x'}
+            padding_inline={'8x'}
+            md={{ padding_inline: '0x', padding_block: '0x' }}
+        >
+            <Flex.Box direction={'col'} justify={'between'} md={{ direction: 'row', gap: '12x' }}>
```
[the remark this answers is not in the document]
Comment (aswathy-deriv) [reply src/features/components/templates/banners/blade-shape-banner/index.tsx:24]: Is it necessary to give inside curly braces?

File: src/features/components/templates/banners/blade-shape-banner/index.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                                padding_block={'8x'}
+                                basis={'6-12'}
+                            >
+                                <Image
+                                    src={item.data.icon}
+                                    alt={item.data.text}
+                                    width={is_mobile ? 24 : 32}
+                                    height={is_mobile ? 24 : 32}
+                                />
+                                <Link pl={'3x'} url={item.data.url} no_hover>
```
Comment (aswathy-deriv) [inline src/features/components/templates/banners/blade-shape-banner/index.tsx:107]: no_hover is for mobile?

File: src/features/components/templates/banners/blade-shape-banner/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hasan-deriv) [reply src/features/components/templates/banners/blade-shape-banner/index.tsx:24]: No, it is not necessary.

**Review by konstantinosG-deriv — APPROVED** [review]

**Review by hasan-deriv — APPROVED** [review]

File: src/features/components/templates/banners/blade-shape-banner/index.tsx (hunk 3 above)
Comment (NikitK-deriv) [reply src/features/components/templates/banners/blade-shape-banner/index.tsx:107]: for everything, but let me check

**Pull request closed without merging**